fluidBuild.config.cjs

```diff
@@ -71,6 +71,7 @@ module.exports = {
 			script: true,
 		},
 		"depcruise": [],
+		"lint:release-tags": [...tscDependsOn, "commonjs"],
```

Comment:
Would it be enough to depend on `tsc` instead of `commonJs`? Or is the `build:test` part of `commonJs` actually necessary before this new one?

Separately: I'm a bit surprised to see that we have many tasks that depend on `[...tscDependsOn, "tsc"]` when `tsc` itself depends on the first set... does anyone know if that's actually by design?